FiniteCocompletion/examples/notebooks/CartesianBraiding.ipynb

```diff
@@ -151,7 +151,7 @@
      "text": [
       "A CAP category with name FiniteStrictProductCompletion( FreeCategory( RightQuiver( \"Q(a,b)[]\" ) ) ):\n",
       "\n",
-      "21 primitive operations were used to derive 81 operations for this category which algorithmically\n",
+      "21 primitive operations were used to derive 84 operations for this category which algorithmically\n",
```

Comment:
I was confused about this change, because I only added 2 operations, but the number here increased by 3. It turns out that the additional operation is due to https://github.com/homalg-project/CAP_project/pull/1227. This was not caught by the CI because `test-notebooks` is (manually?) disabled in the makefile.